### ISSUE TITLE
Training got into NaN values...

### ISSUE THREAD

**Issue by hbzhang** [Justin1904/Low-rank-Multimodal-Fusion#10]
WIth this:

python train_mosi.py --run_id 19260817 --epochs 50 --patience 20 --output_dim 1 --signiture test_run_big_model

I have three pickles files under the data folder. 

I have the following output.                                                          

Model initialized
Epoch 0 complete! Average Training loss: nan
Training got into NaN values...

Model initialized
Epoch 0 complete! Average Training loss: nan
Training got into NaN values...

Model initialized
Epoch 0 complete! Average Training loss: nan
Training got into NaN values...

**Comment by Justin1904** [comment]
Can you check if what you are getting is similar to issue #8? There might've been NaN values in the data that I haven't removed.

**Comment by Bennie61** [comment]
> Can you check if what you are getting is similar to issue #8? There might've been NaN values in the data that I haven't removed.

So where exactly is the NaN value? Delete the data that the NaN value belongs to?Thanks a lot!
